generators/app/templates/WidgetBoilerplate/README.md

```diff
@@ -1,3 +0,0 @@
-# Mendix Widget Boilerplate
```

Comment:
Put back the blank project option. Blank project can just have the project sructure